README.md

```diff
@@ -6,21 +6,122 @@
 
 ## Overview
 
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+[Mailchimp Transactional Email](https://mailchimp.com/developer/transactional/) is a reliable and scalable email delivery service provided by Intuit Mailchimp, designed for sending data-driven transactional emails such as password resets, order confirmations, and notifications.
+
+The `ballerinax/mailchimp.transactional` package provides APIs to connect and interact with the [Mailchimp Transactional API](https://mailchimp.com/developer/transactional/api/), enabling seamless integration with applications to manage transactional email sending, templates, messages, and more.
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Mailchimp Transactional Email connector, you need to have a Mailchimp account and access to a Mandrill API key. Follow the steps below to set up your account and generate the required API key.
+
+### Step 1: Create a Mailchimp Account
+
+1. Go to the [Mailchimp Sign Up Page](https://login.mailchimp.com/signup/) and create a Mailchimp account.
+
+2. Once signed in, click on your profile icon in the bottom-left corner to access the account menu.
+
+    <img src="https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mailchimp/refs/heads/main/docs/setup/resources/mailchimp-dashboard.png" alt="Mailchimp Dashboard" style="width: 70%;"/>
+    <img src=<img src="https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mailchimp/refs/heads/main/docs/setup/resources/mailchimp-menu.png" alt="Mailchimp Profile Menu" style="width: 20%;"/> 
```

Comment:
Use markdown format instead of HTML tags. Check similar places.

```suggestion
    ![Mailchimp dashboard](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mailchimp/refs/heads/main/docs/setup/resources/mailchimp-dashboard.png)
    ![Mailchimp profile menu](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mailchimp/refs/heads/main/docs/setup/resources/mailchimp-menu.png)
```